pypeit/specobjs.py

```diff
@@ -387,34 +129,39 @@ def get_std(self):
 
         """
         # Is this MultiSlit or Echelle
-        pypeline = (self.pypeline)[0]
+        pypeline = (self.PYPELINE)[0]
```

Comment:
Same thing as earlier in term of whether we should be using `in` or `==` below.

Reply:
`in`